policyengine_us/variables/gov/states/md/tax/income/md_income_tax_before_credits.py

```diff
@@ -31,3 +32,13 @@ def formula(tax_unit, period, parameters):
                 p.surviving_spouse.calc(taxable_income),
             ],
         )
+
+        # Add capital gains surtax (starting from specified year)
+        capital_gains_surtax = 0
+        surtax_start_year = parameters(
+            period
+        ).gov.states.md.tax.income.capital_gains.surtax_start_year
+        if period.start.year >= surtax_start_year:
+            capital_gains_surtax = tax_unit("md_capital_gains_surtax", period)
+
+        return regular_income_tax + capital_gains_surtax
```

Comment:
```suggestion
        if period.start.year >= p.capital_gains.surtax_start_year:
            capital_gains_surtax = tax_unit("md_capital_gains_surtax", period)
            return regular_income_tax + capital_gains_surtax

        return regular_income_tax
```
Shorten p earlier in the formula to be usable here